ghdocs/TUTORIAL.md

```diff
@@ -323,24 +507,235 @@ To accomplish this, let's add the following new styles to app.css:
 
 These styles are going to override the default component styling and will help our TextField and Button stand out with some minor positioning adjustments.
 
-Now let's tweak our list a bit. For our purposes, we really want to just have a title and description with the x and checkmark. Since we don't need the paper clip and flag, let's modify one list item to look like the following:
+**Hang on, this part gets a bit tricky**
+
+We really want to tweak our list component for our purposes, we want to have a title and description with the x and checkmark. Since we don't need the paper clip and flag, go ahead and remove all `<div class="ms-ListItem is-selectable"></div>` elements **Except the first one** this will give us one element to work with. 
```

Comment:
I'd rephrase to:
"But now we'd really like to tweak our component by adding a title and a description along with icons for an x and a checkmark. Since we don't need the paper clip and flag, go ahead and remove all `<div class="ms-ListItem is-selectable"></div>` elements **except the first one** so we have one element to work with."